src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -141,10 +141,10 @@ public void setClient(Client target, Client editedClient) {
     }
 
     @Override
-    public boolean hasCountryNote(Country country, Note countryNote) {
-        requireAllNonNull(country, countryNote);
+    public boolean hasCountryNote(CountryNote countryNote) {
+        requireNonNull(countryNote);
 
```

Comment:
I think this could be an unnecessary new line (line 146) or is it that there's supposed to be a blank line before return statements? 🤔

Reply:
ok I removed it to standardise with other methods